apps/client/src/features/rundown/Rundown.tsx

```diff
@@ -160,9 +209,27 @@ export default function Rundown({ data }: RundownProps) {
             reorderEvent(cursor, previousIndex + 1, previousIndex);
           }
         }
+      } else if (modKeysCtrl) {
+        switch (event.code) {
+          case 'KeyC': {
+            event.stopPropagation();
+            setEventCopyId(cursor);
+            break;
+          }
+          case 'KeyV': {
+            event.stopPropagation();
+            if (!eventCopyId) break;
+            const copyEvent = rundown[eventCopyId];
+            if (!isOntimeEvent(copyEvent)) break;
+            const newEvent = cloneEvent(copyEvent, cursor ?? undefined);
+            addEvent(newEvent);
+            setEventCopyId(null);
+            break;
+          }
+        }
       }
     },
-    [order, cursor, rundown, setCursor, insertAtCursor, reorderEvent],
+    [order, cursor, rundown, eventCopyId],
```

Comment:
I'm also a bit unsure about this dependency array stuff
there is no reason for functions to be in here right?

Reply:
Just add what eslint says, that is true in 99% of cases